modules/nodes-cluster-enabling-features-install.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/nodes-cluster-enabling-features.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-cluster-enabling-features-install_{context}"]
+= Enabling feature sets at installation
+
+You can enable feature sets for all nodes in the cluster by editing the `install-config.yaml` file before you deploy the cluster.
+
+.Prerequisites
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Use the `featureSet` parameter to specify the the name of the feature you want to enable. For example, `TechPreviewNoUpgrade`.
```

Comment:
```
For example, `TechPreviewNoUpgrade`.
```

Could be reworked into a full sentence, or combined with the previous sentence? And ending with a colon going into the block, e.g.:

```
[...] specify the name of the feature you want to enable, such as `TechPreviewNoUpgrade`:
```

Also note "the the".